app/views/circle/members/index.slim

```diff
@@ -1,22 +1,48 @@
 h2 = t('.directory')
 
-table
-  tr
-    th = @members.human_attribute_name("name")
-    th = @members.human_attribute_name("circle.roles")
-    th = @members.human_attribute_name("email")
-    th = @members.human_attribute_name("working_groups")
-    th = @members.human_attribute_name("task.completed_tasks")
-    th = @members.human_attribute_name("task.pending_tasks")
-
-  - @members.each do |user|
-    tr
-      td= user.name
-      td= user.circle_roles.for_circle(current_circle).map(&:name).sort.join(", ")
-      td= user.identity.email
-      td
-        - user.working_groups.for_circle(current_circle).each do |wg|
-          li= wg.name
-      td= user.tasks.for_circle(current_circle).volunteered.completed.count
-      td= user.tasks.for_circle(current_circle).volunteered.not_completed.count
+div.table-scrollable
+	table
+	  tr
+	    th = @members.human_attribute_name("name")
+	    th = @members.human_attribute_name("working_groups")
+	    th = @members.human_attribute_name("location.address")
+	    th = @members.human_attribute_name("phone")
+	    th = @members.human_attribute_name("email")
+	    th = @members.human_attribute_name("active_since")
+	    th = @members.human_attribute_name("last_login")
+	    th = @members.human_attribute_name("task.completed_tasks")
+	
+	  - @members.each do |user|
+	    tr
+	      td
+	      	=render partial: 'user/user_icon', locals: {user: user}
+	      	=user.name      
+	      td
+	        - user.working_groups.for_circle(current_circle).each do |wg|
+	          a href= circle_working_group_path(current_circle,wg) 
+	            =wg.name
+	      td= user.location.address
+	      td
+	        - if user.mobile_phone != nil
```

Comment:
nit: prefer [`.present?`](http://api.rubyonrails.org/classes/Object.html#method-i-present-3F) or [`.blank?`](http://api.rubyonrails.org/classes/FalseClass.html#method-i-blank-3F) over nil checks. It tends to be more readable.